SQL/MySQL.sublime-syntax

```diff
@@ -0,0 +1,179 @@
+%YAML 1.2
+---
+name: MySQL
+scope: source.sql.mysql
+version: 2
+extends: Packages/SQL/SQL (basic).sublime-syntax
+
+variables:
+  simple_types: |-
+    (?xi:
+      \b
+      (?:bigint|bigserial|bit|bool|boolean|box|bytea|cidr|circle|date|datetime|double\s+precision|enum|inet|integer|
+        line|longtext|lseg|macaddr|money|ntext|oid|path|point|polygon|real|serial|smallint|sysdate|sysname|text|tinytext|
+        unsigned)
+      \b
+    )
+  types_with_optional_number: |-
+    (?xi:
+      \b
+      (?:bit\s+varying|character\s+(?:varying)?|tinyint|var\schar|float|int|interval|numeric|decimal|times?|timestamp(?:s|tz)?)
+      \b
+    )
+
+contexts:
+  main:
+    - meta_append: true
+    - include: regexps
+
+  comments:
+    - meta_append: true
+    - match: '#'
+      scope: punctuation.definition.comment.sql
+      push: inside-number-sign-comment
+
+  double-dash-comments:
+    - meta_include_prototype: false
+    - match: '--(?=\s)'
+      scope: punctuation.definition.comment.sql
+      push: inside-double-dash-comment
+
+  strings:
+    - meta_append: true
+    - match: '"'
+      scope: punctuation.definition.string.begin.sql
+      push: inside-double-quoted-string
+    - include: begin-interpolation
+
+  inside-double-quoted-string:
+    - meta_include_prototype: false
+    - meta_scope: string.quoted.double.sql
```

Comment:
Haven't done much SQL but I can see some string interpolation contexts. So should we get the `meta.string.sql` context in here to adhere with scope naming guidelines ?

Reply:
good point, I think we should. Though, I'd personally rather leave this for a separate PR. This one is already getting too big - this context hasn't changed from the current one.